src/vmspawn/vmspawn.c

```diff
@@ -259,6 +264,12 @@ static int parse_argv(int argc, char *argv[]) {
                         arg_qemu_gui = true;
                         break;
 
+                case ARG_QEMU_NET:
+                        arg_qemu_net = parse_qemu_network_stack(optarg);
+                        if (arg_qemu_net == _QEMU_NET_INVALID)
```

Comment:
usually we just do a `arg_qemu_net < 0` check here, to catch all kinds of errors

Reply:
Fixed in eae47ee9d9.